### PR TITLE
[bitnami/drupal] Chart standardized

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

This PR standardizes the bitnami/drupal chart to be in line with the rest of the catalog.

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Comment by migruiz4** [comment]
This PR is pending the release of MariaDB standardization major. Please do not merge before it is released and ensure to run `helm dep update`.

**Comment by randradas** [comment]
ok just let me know in order to squash and merge.

**Comment by randradas** [comment]
Not enough context to review this task @javsalgar although it looks good to me but someone else should take a look.

**Review by carrodher — APPROVED** [review]
LGTM

File: bitnami/drupal/values.yaml
Comment (carrodher) [inline bitnami/drupal/values.yaml:431]:
```suggestion
  ##     timeoutSeconds: 300
  ##
  sessionAffinityConfig: {}
```

File: bitnami/drupal/values.yaml
Comment (carrodher) [inline bitnami/drupal/values.yaml:264]:
```suggestion
## @param resources.requests The requested resources for the init container
```

File: bitnami/drupal/values.yaml
Comment (carrodher) [inline bitnami/drupal/values.yaml:264]:
```suggestion
## Drupal container's resource requests and limits
## ref: http://kubernetes.io/docs/user-guide/compute-resources/
## @param resources.requests [object] The requested resources for the init container
## @param resources.limits The resources limits for the init container
```

**Review by carrodher — APPROVED** [review]
LGTM

**Pull request merged**
